src/js/setup-text-tracks.js

```diff
@@ -120,6 +108,13 @@ function attachDashTextTracksToVideojs(player, tech, tracks) {
  * @private
  */
 export default function setupTextTracks(player, tech, options) {
+  // Clear VTTCue if it was shimmed by vttjs and let dash.js use TextTrackCue.
+  // This is necessary because dash.js creates text tracks
+  // using addTextTrack which is incompatible with vttjs.VTTCue in IE11
+  if (window.VTTCue && !(/\[native code\]/).test(window.VTTCue.toString())) {
```

Comment:
Should we also explicitly check for IE/IE11 here using video.js' functions (for further safety)?